Views/AddPersonView.cs

```diff
@@ -0,0 +1,69 @@
+﻿using nightOwl.Models;
+using nightOwl.Presenters;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace nightOwl.Views
+{
+    public partial class AddPersonView : Form, IAddPersonView
+    {
+        private readonly AddPersonPresenter _presenter;
+
+        public AddPersonView(IPersonModel model)
+        {
+            InitializeComponent();
+            _presenter = new AddPersonPresenter(this, model);
+        }
+
+        public void ShowMessage(string message) { MessageBox.Show(message, @"Information", MessageBoxButtons.OK, MessageBoxIcon.Information); }
```

Comment:
Do not put everything in one line.
Why is there `@` when there is no need to have string literal?